components/zoom_admin/actions/delete-webinar-panelist/delete-webinar-panelist.mjs

```diff
@@ -0,0 +1,39 @@
+import zoomAdmin from "../../zoom_admin.app.mjs";
+import get from "lodash/get.js";
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Delete webinar panelist",
+  description: "Remove a panelist for a webinar. [See the docs here](https://marketplace.zoom.us/docs/api-reference/zoom-api/webinars/webinarpanelistdelete)",
+  key: "zoom_admin-action-remove-webinar-panelist",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    zoomAdmin,
+    webinar: {
+      propDefinition: [
+        zoomAdmin,
+        "webinar",
+      ],
+    },
+    panelist: {
```

Comment:
@vellames-turing  I get duplicated panelists listed here: 
![image](https://user-images.githubusercontent.com/20780753/149569161-a62ac658-9a5c-4dae-b3be-174c3e93d6b4.png)

Is this a bug in the component code or within async options code?

Reply:
Fixed!